src/constantGuards.ts

```diff
@@ -1,10 +1,10 @@
 import {ReasonGuard} from './ReasonGuard';
 
-const trueGuard: ReasonGuard<unknown, unknown> = (input, _errs, confs): input is unknown => {
+const trueGuard: ReasonGuard<unknown, unknown> = (input, _errs, confs = []): input is unknown => {
```

Comment:
Why not redefine these the way they are in the tests? Seems nifty.

Reply:
not sure I understand what you mean